charts/agent-control/templates/install-flux.yaml

```diff
@@ -28,6 +28,14 @@ spec:
               cpu: 50m
               memory: 64Mi
           command: ["/bin/sh", "-c"]
+          {{- with (index .Values "agent-control-cd").installer.extraEnv }}
+          env:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          {{- with (index .Values "agent-control-cd").installer.extraVolumeMounts }}
+          volumeMounts:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
```

Comment:
I'm not sure if I've got it right but I think we are defining `volumeMounts` twice 🤔